src/usdb_syncer/usdb_scraper.py

```diff
@@ -287,14 +287,16 @@ def get_usdb_details(song_id: SongId) -> SongDetails:
     html = get_usdb_page(
         "index.php", params={"id": str(int(song_id)), "link": "detail"}
     )
-    return _parse_song_page(BeautifulSoup(html, "lxml"), song_id)
+    return _parse_song_page(BeautifulSoup(html, "lxml"), song_id, logger)
 
 
-def _parse_song_page(soup: BeautifulSoup, song_id: SongId) -> SongDetails:
+def _parse_song_page(
+    soup: BeautifulSoup, song_id: SongId, logger: Log = _logger
```

Comment:
I think we should better address that in the testing code, e.g. by passing a fake Logger like `get_logger(__file__)`.